src/main/java/org/opensearch/sdk/ExtensionSettings.java

```diff
@@ -127,9 +130,11 @@ public ExtensionSettings(
         String hostPort,
         String opensearchAddress,
         String opensearchPort,
+        String routeNamePrefix,
```

Comment:
Personal preference, I like to have all the constructers with fewer args call the one with more args, adding in the defaults. But I can see that wasn't done here so I'm fine with this as-is.